ROS/Chapter_6.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>I notice that bambi is not sampling in parallel for you. Are you using a computer with a single core? Does the same happens with PyMC3? Also are you running 4 chains for some special reason?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Im using docker on my mac laptop which apparently doesnt want to parallel sample. Docker on my linux desktop works fine though. I'm running 4 chains because I thought it was more than 2 but less than 8 :D